examples/index.ts

```diff
@@ -153,6 +192,20 @@ async function doStreams(provider: BaseProvider) {
     console.info(" ")
 }
 
+async function doSolanaStreams(provider: BaseProvider) {
```

Comment:
same here: `SwapStreams` or `AmmStreams`? might be good to rename the original method so it's clear it's orderbookStreams